src/services/formatting/formatting.ts

```diff
@@ -741,7 +743,20 @@ namespace ts.formatting {
                 processNode(child, childContextNode, childStartLine, undecoratedChildStartLine, childIndentation.indentation, childIndentation.delta);
                 if (child.kind === SyntaxKind.JsxText) {
                     const range: TextRange = { pos: child.getStart(), end: child.getEnd() };
-                    indentMultilineCommentOrJsxText(range, childIndentation.indentation, /*firstLineIsIndented*/ true, /*indentFinalLine*/ false);
+                    if (range.pos !== range.end) { // don't indent zero-width jsx text
+                        const siblings = parent.getChildren(sourceFile);
+                        const currentNode = find(siblings, arg => arg.pos === child.pos);
+                        const currentIndex = siblings.indexOf(currentNode!);
+                        const previousNode = siblings[currentIndex - 1];
+                        if (previousNode) {

```

Comment:
I'm... unclear on why this can happen, but `formatTsx` produces a case where you have a `JsxText` node, and it has no previous sibling. Which should be impossible, as it should always have an associated opening tag. The input _does_ have a syntax error, so that's probably why.